00_source_data/data.py

```diff
@@ -256,5 +240,5 @@ def constant_states(states, num_constant_states, df_selection):
     overdose_copy.groupby(["County Code", "Year"])["Deaths"].sum().reset_index()
 )
 
```

Comment:
@EricR401S @lornamariak Are your overdose subsets being synchronized to each other's? As I saw in threshold.py, the used datasets are different and I'm using Lorna's ones to create plotting.

Reply:
@Emmawang00 I don't think so. That data.py file grabs a pre-made overdose_df csv. I don't know the code that created it or modified that csv.

Reply:
@Emmawang00 That's why I'm hesitant to regenerate the subsets, because that csv is ambiguous.

Reply:
I think for overdose death datasets, we could refer to the latest codes created in threshold.py file, while for opioiods prescription datasets, we refer to the datasets here?

Reply:
the overdose csv has data type issues that I fixed in threshold.py. You can take that method and use it.

Reply:
Sounds good!